src/main/java/com/nerodesk/om/aws/AwsAttributes.java

```diff
@@ -47,19 +48,34 @@
  */
 @EqualsAndHashCode
 public final class AwsAttributes implements Attributes {
+    /**
+     * The Doc ocket.
+     */
+    private final transient Ocket ocket;
+
+    /**
+     * Ctor.
+     * @param ockt Ocket
+     */
+    public AwsAttributes(final Ocket ockt) {
```

Comment:
@dmzaytsev actually I think it would be better to use `ObjectMetaData` here, with that, testing of this class becomes easier, and there is no need for todos (see #292)